doc/installation_linux_manual.md

```diff
@@ -0,0 +1,169 @@
+        if not the SDK will use a timer polling approach which is less sensitive for device detection.
+
+2. Run Intel Realsense permissions script from _librealsense2_ root directory:
+   ```sh
+   ./scripts/setup_udev_rules.sh
+   ```
+   Notice: You can always remove permissions by running: `./scripts/setup_udev_rules.sh --uninstall`
+
+3. Build and apply patched kernel modules for:
+   * Ubuntu with Kernel 4.16 <br/>
+     `./scripts/patch-ubuntu-kernel-4.16.sh`
+   * Ubuntu 18/20 with LTS kernel (< 5.13) <br/>
+     `./scripts/patch-realsense-ubuntu-lts.sh`
+   * Ubuntu 20/22 (focal/jammy) with LTS kernel 5.13, 5.15 <br/>
+     `./scripts/patch-realsense-ubuntu-lts-hwe.sh`
+   * Intel® Joule™ with Ubuntu based on the custom kernel provided by Canonical Ltd.  
+     `./scripts/patch-realsense-ubuntu-xenial-joule.sh`
+   * Arch-based distributions
+     1. Install the [base-devel](https://www.archlinux.org/groups/x86_64/base-devel/) package group.
+     2. Install the matching linux-headers as well (i.e.: linux-lts-headers for the linux-lts kernel).
+     3. Patch the uvc module
+        ```sh
+        ./scripts/patch-arch.sh
+        ```
+   * Odroid XU4 with Ubuntu 16.04 4.14 image based on the custom kernel provided by Hardkernel
+     <br/>
+     `./scripts/patch-realsense-ubuntu-odroid.sh`
+   
+   <br/>
+   <details>
+   <summary>What the *.sh script perform?</summary>
+      The script above will download, patch and build realsense-affected kernel modules (drivers).<br/>
+      Then it will attempt to insert the patched module instead of the active one. If failed
+      the original uvc modules will be restored.
+   </details>
+
+   >  Check the patched modules installation by examining the generated log as well as inspecting the latest entries in kernel log:<br />
+       `sudo dmesg | tail -n 50`<br />
+       The log should indicate that a new _uvcvideo_ driver has been registered.  
+       Refer to [Troubleshooting](#troubleshooting-installation-and-patch-related-issues) in case of errors/warning reports.2
+   
+## Building librealsense2 SDK
+
+  * Navigate to _librealsense2_ root directory and run:
+    ```sh
+    mkdir build && cd build
+    ```
+  * Run **ONE** of the following CMake commands:
+    <br/>
+    The default build is set to produce the core shared object and unit-tests binaries in Debug mode. <br/>
+    Use `-DCMAKE_BUILD_TYPE=Release` to build with optimizations.
+    ```sh
+    cmake ../
+    ```
+    Builds _librealsense2_ along with the demos and tutorials:
+    ```sh
+    cmake ../ -DBUILD_EXAMPLES=true
+    ```
+    For systems without OpenGL or X11 build only textual examples:
+    ```sh
+    cmake ../ -DBUILD_EXAMPLES=true -DBUILD_GRAPHICAL_EXAMPLES=false
+    ```
+  * Recompile and install _librealsense2_ binaries:
+    <br/>
+    ```sh
+    sudo make uninstall && make clean && make && sudo make -j$(($(nproc)-1)) install
```

Comment:
Lets leave it as make and add a note about usage of -j for allowing a single core to handle other OS tasks

Reply:
(Only relevant to CPUs with > 1 cores)

Reply:
Done